internal/bminventory/inventory.go

```diff
@@ -77,6 +77,7 @@ const (
 	MediaDisconnected int64 = 256
 	// 125 is the generic exit code for cases the error is in podman / docker and not the container we tried to run
 	ContainerAlreadyRunningExitCode = 125
+	mediaDisconnectionMessage       = "Cannot read from the media (ISO) - media was likely disconnected"
```

Comment:
Doesn't seem like this message should be grouped with those other constants under the same `()`

Reply:
Nevermind, I just realized this message match the exact message in the agent. Please write a comment explaining that this is an API and not simply an arbitrary message.

Reply:
no, This message does not match the message in the agent (the ErrorCode is matched)

Reply:
why?

Reply:
Oh, right, misunderstood. Then my [original comment](https://github.com/openshift/assisted-service/pull/2061/files#r657009527) still applies